installers/legacy-install.sh

```diff
@@ -3,6 +3,8 @@
 #
 # Usage: ./install.sh [-b branch]
 
+set -x
```

Comment:
I guess you were testing something, but we shouldn't land this.

Reply:
Yes, I am still trying to get an understanding of why the mac integration test is failing.

Reply:
It is removed now.  I figured out what it was, the updated `sed` regexp had a non-posix term `\s` which is not supported on the MacOS installation of `sed`.